docs/automerge_synopsis.md

```diff
@@ -0,0 +1,128 @@
+## Automerge Synopsis
+
+[Join the Automerge Slack community](https://communityinviter.com/apps/automerge/automerge)
+
+---
+### Automerge document pages
+* **Automerge synopsis**
+* [Changelog](https://github.com/restarian/automerge/blob/brace_document/docs/changelog.md)
+* [Conflicting changes](https://github.com/restarian/automerge/blob/brace_document/docs/conflicting_changes.md)
+* [Document lifecycle](https://github.com/restarian/automerge/blob/brace_document/docs/document_lifecycle.md)
+* [Examining document history](https://github.com/restarian/automerge/blob/brace_document/docs/examining_document_history.md)
+* [Example usage](https://github.com/restarian/automerge/blob/brace_document/docs/example_usage.md)
+* [Manipulating and inspecting data](https://github.com/restarian/automerge/blob/brace_document/docs/manipulating_and_inspecting_data.md)
+* [Sending and receiving changes](https://github.com/restarian/automerge/blob/brace_document/docs/sending_and_receiving_changes.md)
+* [Text editing support](https://github.com/restarian/automerge/blob/brace_document/docs/text_editing_support.md)
+* Internal data structures
```

Comment:
Can you put something at `docs/internal_data_structures/README.md` so that people can have a nice page with links on it when they visit `docs/internal_data_structures`?

Reply:
The *docs/internal_data_structures* is a directory, not a document. All the documents within that directory will have that relative path.

The internal_data_structures directory can be removed for a flatter look but it seems to categorize nicely.  It could use a jekyll  build for a website though..

Reply:
@restarian Sorry, I wasn’t clear. I meant visiting the directory itself. (i.e. with [this link](https://github.com/restarian/automerge/tree/brace_document/docs/internal_data_structures)).

Reply:
Oh, ya, I see now. It would be nifty to have directory prologue pages. I know there is a few lines of documentation left over for that. 
Perhaps, the platform needs to have an ignore feature so that symlinks can be created which have *README* in the file without altering the documents structure. Other than that, I can place a document file named "readme" in that directory and re-run brace_document as well.